src/storage/PoolConfiguration.sol

```diff
@@ -8,6 +8,7 @@ library PoolConfiguration {
     struct Data {
         bool paused;
         address productAddress;
+        uint256 positionsPerAccountLimit;
```

Comment:
do we want to name this `lpPositionsPerAccountLimit`?
another question would be: do we want to use maker instead of lp?